From 1edd587e2c2154b3e94f5382284c1e193621d7ec Mon Sep 17 00:00:00 2001
From: skyel <razvan.ghitulete@gmail.com>
Date: Sun, 23 Feb 2014 15:17:44 +0100
Subject: [PATCH 09/18] edfi: disable final lto

Since our purpose is not to hyper-optimize, the flto for the
vmlinux causes problems due to the extremely large amount of memory

Signed-off-by: skyel <razvan.ghitulete@gmail.com>
---
 scripts/Makefile.lto | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/scripts/Makefile.lto b/scripts/Makefile.lto
index a801294..f04d97b 100644
--- a/scripts/Makefile.lto
+++ b/scripts/Makefile.lto
@@ -34,18 +34,10 @@ ifeq ($(COMPILER), clang)
 # everything else should tolerate reordering
 	LTO_FINAL_CFLAGS +=-fno-toplevel-reorder
 
-# enable LTO and set the jobs used by the LTO phase
-# this should be -flto=jobserver to coordinate with the
-# parent make, but work around
-# http://gcc.gnu.org/bugzilla/show_bug.cgi?id=50639
-# use as many jobs as processors are online for now
-	LTO_FINAL_CFLAGS := -flto=$(shell getconf _NPROCESSORS_ONLN)
-	#LTO_FINAL_CFLAGS := -flto=jobserver
-
 # Used to disable LTO for specific files (e.g. vdso)
 	DISABLE_LTO := -fno-lto
 
-	LTO_FINAL_CFLAGS += ${LTO_CFLAGS} -fwhole-program
+	LTO_FINAL_CFLAGS += -fwhole-program
 
 ifdef CONFIG_LTO_DEBUG
 	LTO_FINAL_CFLAGS += -dH -fdump-ipa-cgraph -fdump-ipa-inline-details
-- 
1.8.3.1

